components/actionbar/index.css

```diff
@@ -13,7 +13,8 @@ governing permissions and limitations under the License.
 
 :root {
   --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
-  --spectrum-actionbar-padding-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-padding-left: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-padding-right: calc(var(--spectrum-global-dimension-size-200) / 2);
```

Comment:
hmm, not sure if you'd like to use the DNA-vars for that instead of the global dimensions?